src/plugins/plot/axis/YAxis.vue

```diff
@@ -121,20 +121,22 @@ export default {
         },
         setUpYAxisOptions() {
             this.yKeyOptions = this.seriesModel.metadata
-                .valuesForHints(['range'])
-                .map(function (o) {
-                    return {
-                        name: o.name,
-                        key: o.key
-                    };
-                });
+                ? this.seriesModel.metadata
```

Comment:
Can we do the same initial setting of `[]`, then the actual assignment in an if block based on metadata existing?